### PR TITLE
QR code should be white modal

### PR DESCRIPTION
**Pull request by flexsurfer**
QR code should be white modal

### CODE REVIEW

**Review by jeluard — APPROVED** [review]

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/ubhomM for branch PR-4303

<blockquote><div><strong><a href="https://i.diawi.com/ubhomM">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/F5r4xL for branch PR-4303

<blockquote><div><strong><a href="https://i.diawi.com/F5r4xL">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: Makefile [hunk 1]
```diff
@@ -139,6 +139,11 @@ android-ports-avd: ##@other Add reverse proxy to Android Device/Simulator
 	adb -e reverse tcp:3449 tcp:3449
 	adb -e reverse tcp:4567 tcp:4567
 
+android-ports-genymotion: ##@other Add reverse proxy to Android Device/Simulator
```
Comment (rasom) [inline Makefile:142]: is this part of PR's scope?

**Review by rasom — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (flexsurfer) [reply Makefile:142]: yes, to test it you need to run in on android :)

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/io61T4 for branch PR-4303

<blockquote><div><strong><a href="https://i.diawi.com/io61T4">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/TQNs6X for branch PR-4303

<blockquote><div><strong><a href="https://i.diawi.com/TQNs6X">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/Ky9HcD for branch PR-4303

<blockquote><div><strong><a href="https://i.diawi.com/Ky9HcD">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Pull request merged**